forest/benchmarking/direct_fidelity_estimation.py

```diff
@@ -82,10 +124,10 @@ def _exhaustive_dfe(program: Program, qubits: Sequence[int], in_states,
         )
 
 
-def exhaustive_process_dfe(program: Program, qubits: list,
-                           benchmarker: BenchmarkConnection) -> TomographyExperiment:
+def generate_exhaustive_process_dfe_experiment(program: Program, qubits: list,
```

Comment:
good gosh it is ugly. 

maybe we should rethink this?

`gen_xxx_expt` ? Any thoughts?

Reply:
My thoughts exactly ... `gen_XX_expr()`, i.e., `gen_exhaustive_process_dfe_expr()`?

Reply:
@joshcombes Let's come to an agreement about that and make a repo wide PR to fix the API throughout

Reply:
I agree lets poll the interested parties. Not necessary for this PR.